docs/developer-docs/latest/development/backend-customization.md

```diff
@@ -2558,7 +2557,7 @@ _Parameters:_
 
 | Name   | Type   | Description                  |
 | ------ | ------ | ---------------------------- |
-| result | Object | Deleted entry                |
+| result | Object | Array                        | Deleted entry or an array of entries _(in case of bulkDelete)_ |
```

Comment:
Looks like the formatting doesn't like the `|` can you try `Object / Array` instead. I forget what you had originally.

Reply:
![image](https://user-images.githubusercontent.com/8593673/166564067-a042091f-0149-48ab-9efd-c685f563f8e8.png)